pirant/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .utils.RequestHandler import RequestHandler
```

Comment:
Is it good to keep everything here? What if we keep this in app.py and in init.py, I updated as ```from app import DevRant```

Reply:
https://github.com/aayush26/pirant/blob/master/pirant/__init__.py

Reply:
Sure! it's pretty much the same. Usually, if __init__.py will only import app.py, it means that they are equivalent so it can all go in __init__.py. It's very common practice to put code in __init__.py instead of importing it (e.g. https://github.com/requests/requests/blob/master/requests/__init__.py), but it's a matter of taste. I'll change it right away.

Reply:
Thanks. I will read more about init.py too. Will keep your suggestions in mind, and will try to understand which on is better and why.